scripts/versions.sh

```diff
@@ -2,9 +2,21 @@
 # Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 # See the file LICENSE for licensing terms.
 
+set -e
+set -o pipefail
+
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+
+# Pass in the full name of the dependency
+function getDepVersion() {
+    grep -m1 $1 $TELEPORTER_PATH/go.mod | cut -d ' ' -f2
+}
+
 # Set up the versions to be used
-AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v0.2.3'}
```

Comment:
This isn't used anymore

Reply:
We do reference the AWM Relayer version in test.yml. Could we keep this var here, and reference it from that workflow?

Reply:
Sounds good!